environments/linux/docker/compose.yml

```
@@ -72,6 +72,20 @@ services:
 
 ## Configured images ##
 ### Ubuntu 22.04 images:
+  icc-2023:
+    image: "libcudacxx/icc-2022"
+    profiles: ["icc-2022", "all", "base"]
+    tty: true
+    build:
+      context: ../../../
+      dockerfile: environments/linux/docker/icc.base.Dockerfile
+      target: libcudacxx-configured
+      args:
+        ICC_TOOLKIT_VER: "2022.2.1"
+        HOST_CXX: "icx"
```

I believe this is wrong
```suggestion
        HOST_CXX: "icpx"
```